benefits/sentry.py

```diff
@@ -0,0 +1,52 @@
+from benefits import VERSION
+import sentry_sdk
+from sentry_sdk.integrations.django import DjangoIntegration
+import os
+import subprocess
+
+
+SENTRY_ENVIRONMENT = os.environ.get("SENTRY_ENVIRONMENT", "local")
+
+
+# https://stackoverflow.com/a/21901260/358804
+def get_git_revision_hash():
+    return subprocess.check_output(["git", "rev-parse", "HEAD"]).decode("ascii").strip()
+
+
+def get_sha_path():
+    current_file = os.path.dirname(os.path.abspath(__file__))
+    return os.path.join(current_file, "..", "static", "sha.txt")
+
+
+def get_release() -> str:
+    """On 'local' and 'dev', returns the Git SHA. Otherwise, returns the VERSION."""
+
+    sha_path = get_sha_path()
+    if SENTRY_ENVIRONMENT == "local":
+        return get_git_revision_hash()
+    elif SENTRY_ENVIRONMENT == "dev" and os.path.isfile(sha_path):
+        with open(sha_path) as f:
+            return f.read().strip()
+    else:
+        return VERSION
```

Comment:
We're also not married to this; can always change our minds later.

Reply:
This is really cool, I like this approach a lot.

I'm all for always using the SHA to simplify (and, using the `sha.txt` file in all of `dev`, `test`, `prod`).

If there's a way to add arbitrary metadata, it might be nice to have the `VERSION` for the `prod` environment, but otherwise the SHA seems good enough.

Reply:
It's kinda the other way around: https://github.com/cal-itp/benefits/issues/1266. Using the SHA whenever available for now.